external/fv3fit/fv3fit/_shared/predictor.py

```diff
@@ -33,7 +33,15 @@ def load(cls: Type[L], path: str) -> L:
         ...
 
 
-class Predictor(Dumpable):
+class Reloadable(Dumpable, Loadable):
```

Comment:
This is an improvement, though maybe an even easier way is to modify the `predict` method of the `Predictor` so that it can include the graph network? Even if that model's method won't work in the prognostic run. Up to you.

Reply:
I think this way makes it more clear that we're assuming the loaded thing is a Predictor without type-safety. Autoregressive and "predictive" models are different enough to warrant their own concept, and then I'm also probably going to want to have a different API for CycleGAN. Something that does all three of these is probably doing too much.